tests/src/Functional/PrevNextBlockTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Drupal\Tests\localgov_blogs\Functional;
+
+use Drupal\node\NodeInterface;
```

Comment:
Drupal coding standards have changed for imports can this be update to this please

```
use Drupal\Tests\BrowserTestBase;
use Drupal\Tests\node\Traits\NodeCreationTrait;
use Drupal\node\NodeInterface;
```